schema/spacewalk/common/tables/tables.deps

```diff
@@ -248,9 +248,8 @@ suseImageInfoChannel           :: suseImageInfo rhnChannel
 suseImageInfoInstalledProduct  :: suseInstalledProduct suseImageInfo
 suseImageProfile               :: rhnRegTokenChannels web_customer suseImageStore
 suseImageStore                 :: suseCredentials web_customer suseImageStoreType
-suseOSTarget                   :: rhnChannelArch
 suseMinionInfo                 :: rhnServer
-suseServer                     :: rhnServer suseOSTarget
+suseServer                     :: rhnServer
```

Comment:
The `suseServer` table is removed with this PR. I think this whole line could be removed:

```suggestion
```

Reply:
correct